File: Data Structures & Algorithms/max-consecutive-ones/submission-1.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int res=0; int cnt=0;
        for(int num:nums){
            if(num==0){
                res= max(res,cnt);
                cnt=0;
            }
            else{
                cnt++;
            }
        }
        return max(res, cnt);
    }
};

File: Data Structures & Algorithms/majority-element/submission-1.java
class Solution {
    public int majorityElement(int[] nums) {
        
        int res=0, maxcount=0;
        Map<Integer, Integer> mp= new HashMap<>();
        for(int num:nums){
            mp.put(num, mp.getOrDefault(num, 0)+1);
            if(mp.get(num)>maxcount){
                res= num;
                maxcount=mp.get(num);
            }
        }
        return res;
    }
}